Documentation/compatibility.md

```diff
@@ -11,6 +11,8 @@ The Prometheus Operator supports a number of Kubernetes and Prometheus releases.
 
 The Prometheus Operator uses client-go to communicate with Kubernetes clusters. The supported Kubernetes cluster version is determined by client-go. The compatibility matrix for client-go and Kubernetes clusters can be found [here](https://github.com/kubernetes/client-go#compatibility-matrix). All additional compatibility is only best effort, or happens to still/already be supported. The currently used client-go version is "v4.0.0-beta.0".
 
+Due to the use of `CustomResourceDefinitions` Kubernetes >= v1.7.0 is required
```

Comment:
This is generated from the `cmd/po-docgen` tool, we'll need to add this line there as well. And there is a missing colon at the end of the sentence :slightly_smiling_face: .

Reply:
Fixed :)